### DISCUSSION TITLE
Update Versions triggers Auto to run again

### DISCUSSION THREAD

**Discussion by AndrewLeedham** [intuit/auto#2264] [Q&A] [answered]
I feel like I am missing something here, but the "Update Versions" commit that Auto is creating, seems to be running auto again. I would have thought it would have `[skip ci]` in the commit message to prevent this?

Seems like it changed recently but I am not sure why. Is this expected behaviour?

**Comment by AndrewLeedham** [comment]
Seems like https://github.com/intuit/auto/blob/main/plugins/npm/src/index.ts#L531 should include `[skip ci]`?

**Comment by pchalupa** [comment, marked as answer]
If you are using GitHub Actions, you must add the **if** condition to your job.
```yml
    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```
You can find it in an example workflow file https://intuit.github.io/auto/docs/build-platforms/github-actions.
